.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 #### Analyzer package
 
-Example: Microsoft.CodeQuality.Analyzers, version [2.6.0](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.0)
+Example: Microsoft.CodeQuality.Analyzers, version [2.6.1](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.1)
```

Comment:
:memo: This should not include a default value pointing to a valid target. It's difficult for me to go from a diagnostic ID to a package, so a valid value here makes it impossible to distinguish "I don't know" from "I meant this".

Reply:
Thanks, addressed.

Reply:
Revised it further to not require specifying the analyzer name, but just the diagnostic ID.